src/components/Challenges/ChallengeOverview/Overview.js

```diff
@@ -0,0 +1,21 @@
+import { Button, StyleSheet, Text, View } from "react-native";
+import Hr from "../../HorizontalLine/Hr";
+import styles from "./styles";
+
+export default function Overview({ title, description }) {
+  return (
+    <View>
+      <Text style={styles.titleStyles}>{"Challenge Overview"}</Text>
+      <Hr />
+      <View style={styles.todoListContainerStyles}>
+        <Text style={styles.titleStyles2}>{title}</Text>
+        <Text style={styles.descriptionTextSize}>{description}</Text>
+        <View style={styles.startButtonStyles}>
+          <Button color="#1658CD" title="Start" />
```

Comment:
Please use the color Constant in CommonConstants.js file if the color you want is not there define it and use it.

Reply:
done